dali/operators/image/distortion/jpeg_compression_distortion_op_gpu.cu

```diff
@@ -38,17 +37,63 @@ class JpegCompressionDistortionGPU : public JpegCompressionDistortion<GPUBackend
   std::vector<int> quality_;
 };
 
+template <int ndim>
+TensorListShape<ndim - 1> unfold_dim(const TensorListShape<ndim> &shapes) {
+  static_assert(ndim > 1,
```

Comment:
```suggestion
  static_assert(ndim >= 1,
```
We can unfold 1D to a list of scalars.

Reply:
I'd like to prepare separate PR that moves this to utility and widdens scope to handle all cases. Is it fine?

Reply:
Sure, but I think that this assert is simply too strict and changing it won't break anything.

Reply:
Would scalars need if for shape to make it `{1}`, correct me if I am wrong.

Reply:
No. A 0-element shape has a volume of 1 and it just works.